### PR TITLE
Anya/583 record email address

### PR DESCRIPTION
**Pull request by aroltsch**
Connects #583 
1. Add `email` address to the users table
2. Set user's email from the session

Testing:
1. Automated tests

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -71,6 +71,7 @@ def from_session(session)
 
       find_or_create_by(css_id: user["id"], station_id: user["station_id"]).tap do |u|
         u.full_name = user["name"]
+        u.email = user["email"]
```
Comment (amprokop) [inline app/models/user.rb]: Question — how did you verify that `user["email"]` is the right key name to use for the prod/UAT user object? I'd assume it is— did @shanear tell you?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -38,6 +38,8 @@
 
 Capybara.default_driver = ENV["SAUCE_SPECS"] ? :sauce_driver : :sniffybara
 
+ActiveRecord::Migration.maintain_test_schema!
```
Comment (amprokop) [inline spec/rails_helper.rb]: Read about this here: https://relishapp.com/rspec/rspec-rails/docs/upgrade

Excellent!

**Comment by amprokop** [comment]
just one question about how you know that this key name is right!

File: app/models/user.rb (hunk 1 above)
Comment (aroltsch) [inline app/models/user.rb]: From what I understand, in order to login to certification, e-folder handles sign on of a user and stores all the info in the session and then this session is shared between e-folder and certification. In e-folder it is called `email`:
```
def from_css_auth_hash(auth_hash)
      raw_css_response = auth_hash.extra.raw_info
      first_name = raw_css_response["http://vba.va.gov/css/common/fName"]
      last_name = raw_css_response["http://vba.va.gov/css/common/lName"]

      User.new(
        css_id: auth_hash.uid,
        email: raw_css_response["http://vba.va.gov/css/common/emailAddress"],
        name: "#{first_name} #{last_name}",
        roles: raw_css_response.attributes["http://vba.va.gov/css/caseflow/role"],
        station_id: raw_css_response["http://vba.va.gov/css/common/stationId"]
      )
    end
```
@amprokop

Comment (amprokop) [inline app/models/user.rb]: great, thanks 👍

**Pull request merged**